src/services/colorSection.js

```diff
@@ -0,0 +1,37 @@
+export { colorSection, colorSectionInterval }
```

Comment:
`colorSection` is an interesting name. Maybe `selectColor` and `selectColorByInterval`? I couldn't understand what was meant by `colorSection`.

Reply:
Thanks! I changed it accordingly